src/app/work-item/work-item-detail/work-item-detail.component.ts

```diff
@@ -740,8 +740,8 @@ export class WorkItemDetailComponent implements OnInit, AfterViewInit, OnDestroy
 
   selectIteration(iteration: any): void {
     this.selectedIteration = iteration;
-    this.dropdownButton.nativeElement.innerHTML = this.selectedIteration.attributes.name +
-                                                  ' <span class="caret"></span>'
+   // this.dropdownButton.nativeElement.innerHTML = this.selectedIteration.attributes.name +
+     //                                             ' <span class="caret"></span>'
   }
 
```

Comment:
To make area selection functional
(I'm not able to add the comment at the correct line number)
3. Add @ViewChild('areaClick') areaClick: any; before @ViewChild('areaSearch') areaSearch: any;
4. In function activeSearchArea, add this.areaClick.nativeElement.click(); after this.areaSearch.nativeElement.focus();